charter.md

```diff
@@ -0,0 +1,55 @@
+# Interop Team Charter
+
+The Interop Team aims to make the web more interoperable in key areas, prioritized by web developer and user needs. This team is part of [the web-platform-tests project](https://github.com/web-platform-tests/wpt), the main test suite for the web platform.
+
+## Governance
+
+The team makes decisions based on consensus, which is defined as support from at least two participating organizations and no opposition. Decisions can be made both in the repository and in meetings, as long as consensus can be established.
```

Comment:
This is the language we'd like to propose, with the '+' line being my addition to the original text:

```suggestion
The team makes decisions based on consensus, which is defined as support from at least two participating organizations and no opposition. Decisions can be made both in the repository and in meetings, as long as consensus can be established. By default, this process will be public; votes of participating organizations may be shared outside the group. However, by consensus the group may choose in advance to make certain voting and decision-making processes member-confidential.
```

The goal here is to avoid uncertainty or surprises about what votes get will made public, without constraining the group from deciding to make parts of the process member-confidential.

Reply:
This proposed language talks about processes which are not defined in the charter, such as voting, and which probably don’t belong in the charter in the first place, since processes should have the flexibility to be changed by consensus.

At any rate, those things are being worked out on the process side.  Statements about what processes or artifacts should or should not be confidential should be made on the process side, not in the charter.  Therefore, Brian and I believe this doesn’t belong in the charter.

Reply:
@meyerweb I take "this process" in the suggestion to be the process of decision making. Maybe this wording would make it clearer?

> By default, decisions will be public; positions of participating organizations may be shared outside the team. However, by consensus the team may choose in advance to make certain decisions member-confidential.

@dandclark does that match your intended meaning?

Zooming out, I suggest that we decide on this charter wording and whether to make any part of Interop 2024 member-confidental at the same time. I could live with (approve and merge) the above wording and making Interop 2024 Focus Area Selection member-confidential.

Reply:
After discussing and iterating today, I've updated the language to this:

> By default, the group will operate in public. However, specific decision-making processes can be made member-confidential by defining them as such in a process document published by the team.

I'll resolve this discussion, and lean on regular PR review to get that approved by everyone involved.

Reply:
Thanks @foolip , the updated language LGTM.